### PR TITLE
Parse feature

### PR DESCRIPTION
**Pull request by zekth**
ping @bartlomieju

### CODE REVIEW

File: mod.ts [hunk 1]
```diff
@@ -3,8 +3,14 @@ import { readFileStr } from "https://deno.land/std/fs/read_file_str.ts";
 import { StringReader } from "https://deno.land/std/io/readers.ts";
 import { BufReader } from "https://deno.land/std/io/bufio.ts";
 
+export interface HeaderOption {
+  name: string;
+  parse?: (string) => unknown;
```
Comment (j-f1) [inline mod.ts]: This is implicitly of an `any` type. Did you mean this instead?

```suggestion
  parse?: (value: string) => unknown;

File: mod.ts [hunk 2]
```diff
@@ -3,8 +3,14 @@ import { readFileStr } from "https://deno.land/std/fs/read_file_str.ts";
...
+}
+
 export interface ParseOption {
-  header: boolean | string[];
+  header: boolean | string[] | HeaderOption[];
+  parse?: (unknown) => unknown;
```
Comment (j-f1) [inline mod.ts]: Same thing here.

**Review by j-f1 — COMMENTED** [review]
Nice! Added a couple of suggestions.

**Comment by zekth** [comment]
> Nice! Added a couple of suggestions.

Nice thanks for the review! Also eslint does not seem to be triggered by this implicity

EDIT: my bad implicity any is only checked for return type i think

**Comment by zekth** [comment]
Updated the doc. Maybe content some typo.

**Review by bartlomieju — COMMENTED** [review]
Looks good for me, but Travis is red

**Comment by zekth** [comment]
> Looks good for me, but Travis is red

This is due to deno error 134...

**Pull request merged**